sdk/test/AzureTest/Scrubbing.swift

```diff
@@ -0,0 +1,111 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import Foundation
+import DVR
+
+public extension Filter {
+    
+    static let subscriptionIDRegex = "(/(subscriptions))/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
+    static let graphSubscriptionIDRegex = "(https://(graph.windows.net))/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
+    
+    static var subcriptionIDReplacement = "99999999-9999-9999-9999-999999999999"
+    
+    static var standardHeaderScrubbing: [String : FilterBehavior] = [
+        "location": .closure({ key, val in
+            scrubSubscriptionId(from: val ?? "")
+        }),
+        "operation-location": .replace(""),
+        "azure-asyncoperation": .replace(""),
+        "www-authenticate": .replace(""),
+        "access_token": .replace("")
```

Comment:
Should these be replaced with empty string or removed? For the other headers, they were simply removed. Can you find these headers in any of the Python recordings?

Reply:
I wanted to ask you about this. After checking the recording we've been using, none of the above are being included. I'll look into it more but I believe that they should all just be removed so I'll change it.

Reply:
This file is meant to implement the scrubbing things that absolutely should be scrubbed. However it's not enforced in any capacity and someone could improperly use the Filter and record sensitive information. Originally this was why I was writing the standard filter (so that there exists a readily accessible Filter with the appropriate scrubbing), but even that doesn't seem to be enough. What level of protection do you think we need against this? Should we make it not possible to use a filter that doesn't have this scrubbing place?

Reply:
The default for AzureTest should be to use this filter. I might not be understanding the scenario. We can hop on a meeting if needed